docs/site/home.html

```diff
@@ -80,9 +80,18 @@ <h3 class="block-title">What is it?</h3>
       </div>
 
       <div class="quote-box--supplementary">
-        <h4>Retrieve Datastore Entities</h4>
-
-        <div hljs language="js">
+        <h4>Example: Retrieve an entity from Cloud Datastore</h4>
+
+        <div class="code-sample-tabs" ng-init="selected = 'compute engine'">
+          <a ng-click="selected = 'anywhere'"
+             ng-class="{selected: selected == 'anywhere'}"
+             >Run anywhere</a>
```

Comment:
"anywhere" to me suggests anywhere including Compute Engine... I suppose that's correct, this would work anywhere including Compute Engine, but that's not what we are recommending. Maybe "elsewhere" is better (if it's the second tab)? Or keep it this way, that's suffice with me.